src/BUILD.bazel

```diff
@@ -120,5 +121,6 @@ history_server(
         ":copy_material_theme",
         ":copy_systemjs",
         ":copy_zonejs",
+        "//src/images:image_assets",
```

Comment:
Do we need the full path on this “//src”??

Reply:
Yup, we need the full path here. Did you have something else in mind?